portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -49,8 +49,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       long id = entity.getKey().getId();
       String message = (String) entity.getProperty("message");
       long timestamp = (long) entity.getProperty("timestamp");
+      double score = (double) entity.getProperty("score");
```

Comment:
note that along the lines of our discussion, how is this code handling entities where score isn't being set? I assume `getProperty` would return `null` in this case?

Reply:
Ok, so in my most recent commit I did the following:

1) Checked if the score is null
2) Made the score -2 if it is null as this number is impossible (-1 <= score <= 1)

I will show I display this in my next PR!
Here is the updated json file:
![Screen Shot 2020-03-06 at 1 42 41 PM](https://user-images.githubusercontent.com/18583623/76117082-c14aa980-5fb0-11ea-8009-7d6d6e373190.png)